packages/plexus/src/zoom/MiniMap.tsx

```diff
@@ -69,20 +69,20 @@ function getViewTransform(props: TProps, displaySize: { width: number; height: n
   };
 }
 
-function getClassNames(props: TProps) {
+function getClassNames(props: { className: string; classNamePrefix: string }) {
```

Comment:
why not pass as proper args?
```suggestion
function getClassNames(className: string, classNamePrefix: string) {
```